mbl-core/tests/partitions/test_partitions.py

```diff
@@ -126,5 +126,22 @@ def get_actual_part_table():
     return part_table
 
 
+def get_var_for_actual_part(sys_part_info_file):
+    """
+    Get the value of partition information.
+
+    Args:
+    * sys_part_info_file (str): name of file containing the partition
```

Comment:
This looks wrong to me. `sys_part_info_file` is the full path to a file, not just a name.

Additionally, sys_part_info_file isn't a `str` it's a `pathlib.Path`, and the function relies on that.

Reply:
TBH, I don't really get the point of this function. All it's doing is calling `read_text` (except for the exception handling which I don't understand), and I don't think its presence is improving readability, providing useful abstraction, or reducing duplication.

Reply:
Removed the function.